Lib/test/datetimetester.py

```diff
@@ -1336,6 +1336,12 @@ def test_insane_fromtimestamp(self):
             self.assertRaises(OverflowError, self.theclass.fromtimestamp,
                               insane)
 
+    def test_fromtimestamp_with_none_arg(self):
+        # See gh-120268 for more details
+        print(self.theclass)
```

Comment:
```suggestion
		print("Using datetime class:", self.theclass)
```

I know it's a test, but I'd suggest either adding a message for the print or removing it.

Reply:
Oh, this was added by accident 🤦 
Thanks for catching this!